metrics-datadog/src/main/java/org/coursera/metrics/datadog/RegexMetricFilter.java

```diff
@@ -0,0 +1,46 @@
+package org.coursera.metrics.datadog;
+
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricFilter;
+import com.google.common.collect.ImmutableSet;
+
+import java.util.regex.Pattern;
+
+public class RegexMetricFilter implements MetricFilter {
```

Comment:
Seems a little strange we're putting this here vs in the other module. If you put this in the `dropwizard-metrics-datadog` module I don't think you have to explicitly put `guava` in the `pom` since `dropwizard-metrics` comes with `guava`. Did you put it here in case someone would use it with the `metrics-datadog`?